docs/tutorials/blog.md

```diff
@@ -561,7 +561,7 @@ Holy smokes, you did it. You have a blog.
 
 ## Creating Blog Posts
 
-Right now our blog posts (and typo fixes) are tied to deploys. That's gross. The idea here is that your posts would be backed by a database, so we need a way to create a new blog post. We're going to be using actions for that.
+Right now our blog posts (and typo fixes) are tied to deploys. While that works and is a simple way to get started, ultimately we'd like to allow users to create and update posts from within the app itself. The idea here is that your posts would be backed by a database, so we need a way to create a new blog post. We're going to be using actions for that.
```

Comment:
The word "gross" felt a bit too strong here. I agree that it might not be the most user friendly way to require deploys for updates. However, I'd say it is generally a simple and easy way to publish new content.

If you're okay to optimise for markdown- and git-aware users.